recipes/paramspace/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "paramspace" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5a8d171663cfca822bef5688c667f6392f63440b2ba5294be25b0f408654f051
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be noarch: python?

Reply:
`noarch: python` would require that it can run on _any_ Python version, right? While the package does not distinguish between Python versions, it does require Python >= 3.6. Can that requirement be made compatible with `noarch: python`?

Reply:
You can put a version constraint on Python below:  `- python >=3.6`

Reply:
noarch python just means that is doesn't ship any extension modules and that none of its deps are truly platform specific

Reply:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

Reply:
You can see more information about ``noarch`` packages here
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python